docs/source/contribution.rst

```diff
@@ -431,15 +431,22 @@ To build the documentation, you need to install `Sphinx <http://www.sphinx-doc.o
 
   $ pip install sphinx sphinx_rtd_theme
 
-Then you can build the documentation in HTML format locally::
+Docstrings (documentation comments in the source code) are collected from the installed Chainer module.
+If Chainer has not been installed yet, then ``make html`` will yield an error that the reference sources cannot
+be read.
+
+Another reason it may be necessary to reinstall Chainer is when you have edited docstrings and want to see those changes reflected in the generated html. In that case,  make sure to install the module before building the
```

Comment:
This sentence sounds a bit unexpected to me, because there's no mention about "reinstalling" above.
Actually if Chainer is installed with `-e` option as demonstrated below, I think no reinstalling is needed.
Am I missing something?

Reply:
Thanks for the comment. I've edited to make more clear. PTAL!